guide/private-key-management/external-signer.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: guide
+title: External signers
+description: What External signers are and their role in securing bitcoin.
+nav_order: 4
+parent: Private key management
+permalink: /guide/private-key-management/external-signers/
+image: https://bitcoin.design/assets/images/guide/private-key-management/external-signers/external-signing-device-preview.png
+redirect_from:
+ - /guide/private-key-management/external-signing-device/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Description of what an external signers are and what the scheme consists of.
+
+Illustration sources
+
+https://www.figma.com/community/file/888680264445459448
+https://www.figma.com/community/file/995256542920917246/BDG---Private-key-management-illustrations
+https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=261%3A1837
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/private-key-management/external-signers/external-signing-device.png"
+   retina = "/assets/images/guide/private-key-management/external-signers/external-signing-device@2x.png"
+   mobile = "/assets/images/guide/private-key-management/external-signers/external-signing-device-mobile.png"
+   mobileRetina = "/assets/images/guide/private-key-management/external-signers/external-signing-device-mobile@2x.png"
+   alt-text = "Examples of popular external signers."
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# External signers
+
+External signers use, store, and manage private keys and other sensitive info, such as the user's recovery phrase, externally from the user's wallet application. The most common external signers used today are hardware wallets. Software-only external signing applications also exist, but are less secure than their hardware counterparts. 
+
+A wallet application supporting external signers removes the need for storing private keys locally by the wallet application. This decreases the theft risk as the keys can be isolated from the internet and/or a potentially malicious device. 
+
+### How it works
+
+<div class="center" markdown="1">
+
+The external signer generates and stores private keys externally, and preferably offline, from the users primary wallet application.
+
+To sign a transaction, the primary wallet application submits a partially signed bitcoin transaction ([PSBT](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki)) to the external signer. 
+
+The external signer then signs the PSBT and sends it back to the primary wallet application. The users primary wallet application now broadcasts this now signed transaction to the Bitcoin network for processing. This process can also happen fully air-gapped using QR codes or memory cards instead of cables or wireless connections, to keep the device offline at all times.
```

Comment:
```suggestion
The external signer then signs the transaction or PSBT and sends it back to the primary wallet application. The users primary wallet application now broadcasts this now signed transaction to the Bitcoin network for processing. This process can also happen fully air-gapped using QR codes or memory cards instead of cables or wireless connections, to keep the device offline at all times.
```